packages/typespec-test/test/openai_generic/tspconfig.yaml

```diff
@@ -4,7 +4,7 @@ options:
   "@azure-tools/typespec-ts":
     azureSdkForJs: false
     isModularLibrary: true
-    "emitter-output-dir": "{project-root}/../generated/typespec-ts"
+    "emitter-output-dir": "{project-root}/generated/typespec-ts"
```

Comment:
I vaguely remember this was trying to test against some output folder issue, thought I am not sure if it's still value to keep it.